code/DDSCodeTester.cpp

```diff
@@ -416,6 +416,31 @@ void dds_domain_examples()
         //!--
     }
 
+    {
+        //DDS_CHANGE_DOMAINPARTICIPANTEXTENDEDQOS
+        // Create a DomainParticipant with default DomainParticipantExtendedQos
+
+        DomainParticipantExtendedQos extended_qos;
```

Comment:
Here, instead of only defining the DomainParticipantExtendedQoS, it would be nice having an example of how to use `get_participant_extended_qos_from_profile()`, retrieve the DomainParticipantExtendedQos then changing something in the DomainParticipantExtendedQos and in the end creating a participant.